website/docs/docs/collaborate/git/setup-azure.md

```diff
@@ -114,8 +114,13 @@ A service user account must have the following Azure DevOps permissions for all
 
 Some of these permissions are only accessible via the Azure DevOps API, for which documentation can be found [here](https://docs.microsoft.com/en-us/azure/devops/organizations/security/namespace-reference?view=azure-devops). We’ve also detailed more information on Azure DevOps API usage below to help accelerate the set up. Alternatively, you can use the Azure DevOps UI to enable permissions, but you cannot get the least permissioned set.
 
-:::info  Provide the service user with required permissions before setting up a dbt Cloud project
-This service user's permissions will also power which repositories a team can select from during dbt project set up, so an Azure DevOps admin must grant at minimum Project Reader access to the service user before setting up a project in dbt Cloud.
+:::info  Provide the service user with required permissions before creating a new dbt Cloud project
+This service user's permissions will also power which repositories a team can select from during dbt project set up, so an Azure DevOps admin must grant at minimum Project Reader access to the service user before creating a new project in dbt Cloud. If you are migrating an existing dbt project to use the native Azure DevOps integration, the dbt Cloud account's service user must have proper permissions on the repository before migration.
```

Comment:
is [this](https://learn.microsoft.com/en-us/azure/devops/organizations/security/permissions?view=azure-devops&tabs=preview-page) a good link for the user to review permissions?

Reply:
I use this [link](https://learn.microsoft.com/en-us/azure/devops/organizations/security/namespace-reference?view=azure-devops) 

I reference it in this [part of our docs](https://docs.getdbt.com/docs/collaborate/git/setup-azure#more-on-service-users)